core/vidl/vidl_ffmpeg_convert.h

```diff
@@ -46,5 +47,26 @@ vidl_pixel_format_from_ffmpeg(PixelFormat ffmpeg_pix_fmt);
 PixelFormat
 vidl_pixel_format_to_ffmpeg(vidl_pixel_format vidl_pix_fmt);
 
+#else
+//: Convert the pixel format of a frame using FFMPEG
+//
+// The \p in_frame->data() is converted from \p in_frame->pixel_format()
+// to \p out_frame->pixel_format() and stored in \p out_frame->data()
+// If the output frame data is not the correct size new memory
+// will be allocated
+bool vidl_ffmpeg_convert(const vidl_frame_sptr& in_frame,
+                               vidl_frame_sptr& out_frame);
+
+
+//: Find the vidl pixel format that matches a FFMPEG one
+vidl_pixel_format
+vidl_pixel_format_from_ffmpeg(AVPixelFormat ffmpeg_pix_fmt);
```

Comment:
Ah, now I see that it's the AVPixelFormat type that has changed.  Can we typedef or define PixelFormat to AVPixelFormat for older versions to avoid the code duplication?